packages/docs/docs/fhir-basics.md

```diff
@@ -15,25 +36,28 @@ Medplum stores healthcare data using the FHIR standard. Storing data according t
 
 While FHIR is quite powerful, it can have a bit of a learning curve. The page will go over the basic concepts for understanding FHIR data in Medplum. For more information, you can check out the [official FHIR documentation](http://hl7.org/fhir/).
 
-## Resources
-
-A core data object in FHIR is called a [**Resource**](https://www.hl7.org/fhir/resource.html). You can think of Resources as **objects** in object oriented languages. The FHIR standard defines a set of [**Resource Types**](./api/fhir/resources), similar to **classes**, that have been built for healthcare applications.
+## Storing Data: Resources
 
-Resources can represent a broad range of healthcare ideas, from very **concrete healthcare items** (e.g. [Patient](./api/fhir/resources/patient), [Medication](./api/fhir/resources/medication), [Device](./api/fhir/resources/device)) or more **abstract concepts** (e.g. [Procedure](./api/fhir/resources/procedure), [Care Plan](./api/fhir/resources/careplan), [Encounter](./api/fhir/resources/encounter)).
+A core data object in FHIR is called a [**Resource**](https://www.hl7.org/fhir/resource.html). You can think of Resources as **objects** in object oriented languages.
 
-A `Resource` is composed of multiple fields, called `Elements`, each of which can be a **primitive type** (e.g. strings, numbers, dates) or a **complex type** (e.g. JSON objects).
+The FHIR standard defines over 150 [**Resource Types**](./api/fhir/resources) that model broad range of healthcare-specific concepts. These include **concrete entities** ([`Patient`][patient], [`Medication`][medication], [`Device`][device]) as well as **abstract concepts** ([`Procedure`][procedure], [`CarePlan`][careplan], [`Encounter`][encounter]).
 
-### Example
+Each field in a resource is called an [**Element**](https://hl7.org/fhir/R4/element.html), each of which can be a **primitive type** (e.g. `string`, `number`, `date`) or a **complex type** (e.g. [`HumanName`](/docs/api/fhir/datatypes/humanname)).
 
-The example below shows an example **[Patient](./api/fhir/resources/patient) resource**, represented as JSON. Here we can see that the `Patient` contains multiple `Elements`, including: `name`, `telecom`, and `address`.
+Lastly, all resources have an `id` element, which is a server-assigned identifier that serves as their **primary key**.
 
+<details>
+<summary>
+Example [`Patient`][patient]
```

Comment:
I'm concerned that hiding this content inside a `<details>` tag will negatively impact our SEO.

(j/k)